packages/form-js-editor/src/render/components/properties-panel/entries/ValuesSourceEntries.js

```diff
@@ -0,0 +1,219 @@
+import { get, isUndefined, without } from 'min-dash';
+
+import { SelectEntry, isSelectEntryEdited, TextFieldEntry } from '@bpmn-io/properties-panel';
+import ValueEntry from './ValueEntry';
+
+import { arrayAdd } from '../Util';
+import { useService } from '../../../hooks';
+
+export function ValuesSourceSelectorEntry(props) {
+  const {
+    editField,
+    field
+  } = props;
+
+  return [ {
+    id: 'values-source',
+    component: ValuesSourceSelect,
+    isEdited: isSelectEntryEdited,
+    editField,
+    field,
+  } ];
+}
+
+export function InputKeyValuesSourceEntry(props) {
+  const {
+    editField,
+    field
+  } = props;
+
+  return [{
+    id: 'input-values-key',
+    component: InputValuesKey,
+    label: 'Input values key',
+    description: 'Define which input property to populate the values from',
+    isEdited: isSelectEntryEdited,
+    editField,
+    field,
+  }];
+}
+
+export function StaticValuesSourceEntry(props) {
+  const {
+    editField,
+    field,
+    id: idPrefix
+  } = props;
+
+  const {
+    values
+  } = field;
+
+  const addEntry = (e) => {
+
+    e.preventDefault();
+
+    const index = values.length + 1;
+
+    const entry = {
+      label: `Value ${index}`,
+      value: `value${index}`
+    };
+
+    editField(field, VALUES_SOURCES_PATHS[VALUES_SOURCES.STATIC], arrayAdd(values, values.length, entry));
+  };
+
+  const removeEntry = (entry) => {
+    editField(field, VALUES_SOURCES_PATHS[VALUES_SOURCES.STATIC], without(values, entry));
+  };
+
+  const validateFactory = (key) => {
+    return (value) => {
+      if (value === key) {
+        return;
+      }
+
+      if (isUndefined(value) || !value.length) {
+        return 'Must not be empty.';
+      }
+
+      const isValueAssigned = values.find(entry => entry.value === value);
+
+      if (isValueAssigned) {
+        return 'Must be unique.';
+      }
+    };
+  };
+
+  const items = values.map((entry, index) => {
+    const id = idPrefix + '-staticValue-' + index;
+
+    return {
+      id,
+      label: entry.label,
+      entries: ValueEntry({
+        editField,
+        field,
+        idPrefix: id,
+        index,
+        validateFactory
+      }),
+      autoFocusEntry: true,
```

Comment:
```suggestion
      autoFocusEntry: id + '-label'
```

Otherwise, it will always focus on the _first input of the page_.